### PR TITLE
[PLAYER-21] Added accent color config parameter

### PR DESCRIPTION
**Pull request by nagavenivn**

### CODE REVIEW

**Comment by nagavenivn** [comment]
@mistermendez  Could you please do the CR?

**Comment by mistermendez** [comment]
should we also update scrubberBar.scrubberHandleColor?

**Comment by lental** [comment]
hey, so you are setting some values to empty string, and instead the value of accentColor is used?

If someone were to fill in those empty strings, those would override accentColor.  is that correct?

**Comment by nagavenivn** [comment]
@lental Yes, that's correct.  Individual settings will override accentColor settings.

@mistermendez scrubberBar.scrubberHandleColor. I am not sure what it supposed to do? Is it playhead color? I didn't see anywhere this is used in the code.

**Comment by mistermendez** [comment]
yes it's playhead color, it appears to be using scrubberBar.playedColor.

**Comment by lental** [comment]
@nagavenivn, scrubberHandleColor is a new feature, and not yet used in html5-skin.  that's why you don't see it used in the code yet

**Comment by lental** [comment]
@pcaseooyala  FYI on some changes in the skin that we will need to be aware of

**Comment by nagavenivn** [comment]
@mistermendez Discussed with Steve, as per his recommendation as of now we will not be using this for changing the playhead color. We will use the played color for the playhead.  I will mention the same in the ticket as well for tracking purpose.

File: skin.json [hunk 1]
```diff
@@ -147,11 +148,11 @@
   },
   "controlBar": {
     "volumeControl": {
-      "color": "rgba(67,137,255,1)"
+      "color": ""
     },
     "iconStyle": {
       "active": {
-        "color": "#FFFFFF",
```
Comment (nagavenivn) [inline skin.json]: @mistermendez As discussed with you over the chat, this was the requirement from Steve and David

**Comment by mistermendez** [comment]
👍

**Pull request merged**